frontend/src/app/Settings/ManageOrganization.tsx

```diff
@@ -11,15 +11,16 @@ interface Props {
 }
 
 const ManageOrganization: React.FC<Props> = (props) => {
-  const [name, setName] = useState(props.name);
+  const [name, setName] = useState("");
   const [formChanged, setFormChanged] = useState(false);
 
   useEffect(() => {
```

Comment:
Do we need this `useEffect` at all? It looks like we're keying off of the org's `name` prop, and any change to that should cause a re-render. I wonder if we should be using the name tho, seems like the unique id would be more correct since users can edit the name. The `onChange` for the name text input should set the `formChanged`.

Reply:
the component first renders when props has the name `name = ""`, and then this component re-renders with `name=<orgName>`.  The state doesn't update on the second re-render with the org name.

The `formChanged` might not need to be in useEffect